students/chan/westagram/users/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db        import models
+from django.db.models import Model
+
+# Create your models here.
+class User(models.Model):
+    name                  = models.CharField(max_length=45)
+    email                 = models.EmailField(max_length=128, null=False, unique=True)
+    password              = models.CharField(max_length=200, null=False)
+    phon_number           = models.CharField(max_length=11, null=False)
+    business_number       = models.CharField(max_length=50, blank=True)
+    allowing_email_recive = models.BooleanField(default=True)
+    enterprize            = models.BooleanField(default=True)
```

Comment:
```suggestion
    allowing_email_recive = models.BooleanField(default=True)
    enterprize            = models.BooleanField(default=True)
```
1. enterpize의 역할이 무엇인가요?? (궁금)
2. 보통 boolean을 사용할 경우 앞에 is_ 를 붙여서 이름을 만듭니다.
3. 데이터의 이력 관리를 위해서 created_at, updated_at column을 추가해주세요.

```python
created_at = models.DateTimeField(auto_now_add=True)
updated_at = models.DateTimeField(auto_now=True)
```

Reply:
1. eterprize의 역할은 기업회원인지 구분하기 위해서 넣었습니다.
2. boolean을 사용하기 위해서 앞에 is 추가 완료 하였습니다.
3. 데이터 이력관리를 위한 created_at, updated_at column 추가 완료하였습니다.

Reply:
models.py 에서 오타 추가 수정하였습니다.